test/publishing_api/special_route_publisher_test.rb

```diff
@@ -40,12 +40,18 @@
             ],
             publishing_app: special_route[:publishing_app],
             rendering_app: special_route[:rendering_app],
-            update_type: "major",
             public_updated_at: Time.now.iso8601,
           }
-        )
+  }
 
-        publisher.publish(special_route)
+  let(:ok_response) { OpenStruct.new({code: 200}) }
```

Comment:
Perhaps we can user the api test helpers to replace the expectations on the `publishing_api` object:

```ruby
it "publishes the special routes" do
  stub_any_publishing_api_call

  publisher.publish(special_route)

  assert_publishing_api_put_content(....)
  assert_publishing_api_publish(...)
end
```

This feels more in line with how normal calls are tested, and would remove the need for returning mock OpenStruct objects.